src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -51,9 +51,15 @@ public void pull() {
     spinWheels(isConeMode ? kConeIntakeSpeed : kCubeIntakeSpeed);
   }
 
-  public void setMode(boolean isConeMode) {
+  /*public void setMode(boolean isConeMode) {
     this.isConeMode = isConeMode;
   }
+*/
+    public void setConeMode() {
+  }
```

Comment:
We need this method to actually change the value for this.isConeMode

For example here it would be like:
this.isConeMode = true;
in between line 58 and 59